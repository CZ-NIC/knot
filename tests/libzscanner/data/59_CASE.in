$ORIGIN	.
$TTL	1

; OK
@	CASE	0	"CZ"	A	127.0.0.1
@	CASE	0	"CZ"	AAAA	::1
@	CASE	0	"CZ"	TXT	"test" "strings"
@	CASE	255	"CZ"	A	127.0.0.1	; Comment
@	CASE	1	"192.168.0.0/24"	CNAME	example.com.
@	TYPE65333	\# 10 0102435A00017F000001
@	TYPE65333	0	""	A	127.0.0.1
@	case	0	""	A	127.0.0.1

; KO
@	CASE
@	CASE						; Empty rdata
