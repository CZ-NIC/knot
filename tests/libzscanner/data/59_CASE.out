OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=0002435A00017F000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=0002435A001C00000000000000000000000000000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=0002435A0010047465737407737472696E6773
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=FF02435A00017F000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=010E3139322E3136382E302E302F32340005076578616D706C6503636F6D00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=0102435A00017F000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=000000017F000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FF35
RDATA=000000017F000001
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
